packages/avatars/demo/avatar.stories.mdx

```diff
@@ -24,13 +24,25 @@ import { AVATAR_TYPE as TYPE } from './stories/data';
       badge: { control: 'number' },
       foregroundColor: { control: 'color' },
       children: { control: 'text', table: { category: 'Avatar.Text' } },
+      status: {
+        options: STATUS_TYPE,
+        control: { type: 'select' }
+      },
+      badgeType: {
+        options: BADGE_TYPE,
+        control: { type: 'select' },
+        name: 'badge type',
+        table: { category: 'Story' }
+      },
+      badgeChildren: { name: 'badge children', control: 'text', table: { category: 'Story' } },
       type: {
         control: 'radio',
         options: TYPE,
         table: { category: 'Story' }
       }
     }}
     parameters={{
+      layout: 'centered',
```

Comment:
Why was this added? Demos are default layout, unless `placement` is used to demonstrate orientation.